CMakeLists.txt

```diff
@@ -71,11 +71,15 @@ if (${TARGET_32BIT})
     INSTALL_COMMAND ""
     BUILD_ALWAYS TRUE)
 
-  ExternalProject_Get_Property(SymRuntime32 BINARY_DIR)
-  set(SYM_RUNTIME_32BIT_DIR ${BINARY_DIR})
+  ExternalProject_Get_Property(SymccRuntime32 BINARY_DIR)
+  set(SYMCC_RUNTIME_32BIT_DIR ${BINARY_DIR})
 endif()
 
-find_package(LLVM REQUIRED CONFIG)
+find_package(LLVM QUIET ${LLVM_VERSION} REQUIRED CONFIG)
```

Comment:
It makes me a little uncomfortable that users now _have to_ specify a version, where previously CMake would have picked any version that it could find. Should we default to the old behavior if the user doesn't explicitly specify a version? I think it should be possible to do so by not passing a version argument to `find_package` in that case.

Then it probably makes sense to forward whatever LLVM version we end up using in SymCC to the runtime, so that they both use the same version.

Reply:
Sounds good. I'll apply the change.